### ISSUE TITLE
Can't login with seeded admin account

### ISSUE THREAD

**Issue by marcelgruber** [rappasoft/laravel-5-boilerplate#64]
I ran the setup and everything worked fine, but when I went to login as the admin, it said that the credentials were not recognized. I also registered other accounts which worked fine, but it's as if the accounts created on the front end are in another database table altogether. When I do SELECT * FROM USERS; it shows the two default seeded accounts, but not any of the accounts I created using the web interface. Stranger still, I can't even seem to find out where those accounts I created are being stored in the database. They certainly aren't in the users table, but I can still somehow log in with them! Thoughts? I know this has to be some stupid mistake or misunderstanding.

**Comment by rappasoft** [comment]
That makes no sense at all. Are you sure you've refreshed?

Anthony Rappa

**Comment by rappasoft** [comment]
There should only be one table for users. Did you go through the whole database?

Anthony Rappa

**Comment by marcelgruber** [comment]
When I refresh the database (therefore clearing out the seeds and what SHOULD be the new accounts I created), the new accounts I created still work. This clearly means that the database I'm refreshing / seeding and the database that I'm actually using are completely different. There must be some kind of mixup somewhere, but it's hard to figure out how.

When I SSH into the vagrant box and enter MYSQL, SHOW DATABASES lists the following dbs:

- information_schema
- homestead
- mysql
- performance_schema

When going into homestead, I see all of the typical tables associated with this app, but once again it seems as though this database isn't actually being used by the app. There is another database somewhere that it's using instead.

Marcel Gruber
